### PR TITLE
Fix include-path warnings

### PR DESCRIPTION
**Pull request by bremoran**
Fix warnings caused by ARMmbed/mbed-drivers#104

### CODE REVIEW

**Comment by bremoran** [comment]
cc @bogdanm @meriac @0xc0170 @autopulated @niklas-arm

**Comment by bogdanm** [comment]
+1

**Comment by meriac** [comment]
++

File: source/pwmout_api.c [hunk 1]
```diff
@@ -174,7 +174,7 @@ void pwmout_period_us(pwmout_t* obj, int us)
     SystemCoreClockUpdate();
 
     TimHandle.Init.Period        = us - 1;
-    TimHandle.Init.Prescaler     = (uint16_t)(SystemCoreClock / 1000000) - 1; // 1 �s tick
+    TimHandle.Init.Prescaler     = (uint16_t)(SystemCoreClock / 1000000) - 1; // 1 �s tick
```
Comment (AlessandroA) [inline source/pwmout_api.c]: (pedantic mode: ON) this weird character here was a `µ`; probably a `u` is better

Comment (bremoran) [inline source/pwmout_api.c]: You are quite correct!  Due to a quirk of github, I didn't see this comment until after the merge.

**Pull request merged**